src/main/java/banking/account/controller/BankController.java

```diff
@@ -0,0 +1,39 @@
+package banking.account.controller;
+
+import banking.account.domain.BasicAccount;
+import banking.account.dto.AccountDTO;
+import banking.account.service.BankService;
+import banking.account.service.Service;
+
+import java.math.BigDecimal;
+
+
+public class BankController {
+    // 싱글톤 패턴
+    public static final BankController bankController = new BankController();
+    private Service bankservice = BankService.getInstance();
+
+    public static BankController getInstance() { return bankController; }
+
+    public BasicAccount findAccount(String accountNumber) {
+        return bankservice.findAccount(accountNumber);
+    }
+
+    public void createAccount(AccountDTO accountDTO) {
+        bankservice.createAccount(accountDTO);
+    }
+
+
+    public void deposit(String accountNum, BigDecimal depositAmount) {
+        bankservice.deposit(accountNum, depositAmount);
+    }
+
+    public void withdraw(String accountNum, BigDecimal depositAmount) {
```

Comment:
```suggestion
    public void withdraw(String accountNum, BigDecimal withdrawAmount) {
```
매개변수의 변수명을 적절하게 지어줘야 할 것 같습니다